src/systemctl/systemctl.c

```diff
@@ -1835,12 +1835,10 @@ static const struct bus_properties_map machine_info_property_map[] = {
 };
 
 static void machine_info_clear(struct machine_info *info) {
-        if (info) {
```

Comment:
not that it matters at all, but maybe an assert() would be good here in cases like this